std/algorithm/mutation.d

```diff
@@ -2419,3 +2462,11 @@ void uninitializedFill(Range, Value)(Range range, Value value)
         // Doesn't matter whether fill is initialized or not
         return fill(range, value);
 }
+
+///
+@safe nothrow unittest
+{
+    auto s = new int[5];
```

Comment:
Are you sure this is equivalent to the original example? IIRC the GC will initialize memory, but `malloc` will not, so there's a difference here.